### ISSUE TITLE
New configuration format

### ISSUE THREAD

**Issue by frcroth** [eisen-oxid/thermit#49]
Large .env files might become bad to handle, there are no hierarchies etc.

Perhaps there are better formats for configuration files.

**Comment by frcroth** [comment]
.toml, .yaml ( Semantic indentation 🤢) , .json (No comments 😞)

**Comment by sirkrypt0** [comment]
> .toml, .yaml ( Semantic indentation nauseated_face) , .json (No comments disappointed)

I agree on this issue! I like tomls so we can go that way if you agree.

**Comment by frcroth** [comment]
Yes toml it is then
